### PR TITLE
Fix Vet Tec wrapping text

### PR DESCRIPTION
**Pull request by NateJohnson-BAH**
## Description
On the VET TEC profile page, the "(25%)" after "Upon enrollment in program" and the one after "Upon completion of program" are wrapping to the next line. The percentage should be displayed in-line.

https://app.zenhub.com/workspaces/vft-59c95ae5fda7577a9b3184f8/issues/department-of-veterans-affairs/vets.gov-team/19578

## Testing done
confirmed behavior in local UI at various scales

## Screenshots
Labels now appear inline at 100% scale, but change behavior based on reactive scale properties. 
Screen Shot 2019-09-03 at 3 30 27 PM

## Acceptance criteria
- [x] The percentage should be displayed in-line.

## Definition of done
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Review by shawkey-daniel — APPROVED** [review]
LGTM

File: src/applications/gi/components/vet-tec/VetTecCalculator.jsx [hunk 1]
```diff
@@ -10,6 +10,7 @@ import {
 import { getCalculatedBenefits } from '../../selectors/vetTecCalculator';
 import VetTecCalculatorForm from './VetTecCalculatorForm';
 import PropTypes from 'prop-types';
+import environment from '../../../../platform/utilities/environment';
```
Comment (shawkey-daniel) [inline src/applications/gi/components/vet-tec/VetTecCalculator.jsx]: You can use the alias for platform: 'platform/utilities/environment'

**Review by shawkey-daniel — CHANGES_REQUESTED** [review]
Reviewers have preferred to use a ternary for production flags.

**Review by shawkey-daniel — APPROVED** [review]
LGTM

**Review by delli-gatti-michael — APPROVED** [review]
Approved by QA

**Review by rianfowler — APPROVED** [review]

**Pull request merged**